packages/ui/src/theme/tokens/space.js

```diff
@@ -7,4 +7,14 @@ module.exports = {
   xl: { value: '2.0rem' },
   xxl: { value: '3.0rem' },
   xxxl: { value: '4.5rem' },
+  relative: {
```

Comment:
What's the intended use of these, they appear to have all the same values as the non-relative sizes?

Reply:
the difference is the units, `rem` vs `em` so that we can size relative to the parent instead of the root.

Reply:
just creating a set of sizes using `em` instead of `rem` for sizing based off of the parent instead of the root. These are being used for resizing the switchfield with a single rule on the parent element instead of placing a data-size attribute/rule on every piece that needs to be resized.

Reply:
added a comment to the code